src/librustdoc/Cargo.toml

```diff
@@ -13,6 +13,9 @@ env_logger = { version = "0.4", default-features = false }
 log = "0.3"
 pulldown-cmark = { version = "0.0.14", default-features = false }
 
+[target.'cfg(not(any(stage0, stage1)))'.dependencies]
+html-diff = "0.0.3"
+
```

Comment:
Not sure. I don't know exactly what exactly changed since the build update. I'll look in a few hours.